engine/src/mblflst.cpp

```diff
@@ -80,6 +80,29 @@ MCFontnode::MCFontnode(MCNameRef fname, uint2 &size, uint2 style)
 #endif
 }
 
+#ifdef TARGET_SUBPLATFORM_IPHONE
+extern bool coretext_get_font_name(void*, MCNameRef&);
+extern uint32_t coretext_get_font_size(void*);
+
+MCFontnode::MCFontnode(MCSysFontHandle p_handle, MCNameRef p_name)
+{
+    if (p_name == nil)
+        coretext_get_font_name(p_handle, &reqname);
+    else
+        reqname = MCValueRetain(p_name);
```

Comment:
Ah, the OSX code that it was copied from uses a plain MCNameRef instead of an auto-variant. That'll teach me to copy-paste code I wrote months ago!